toolbox/githubctl/main.go

```diff
@@ -24,14 +24,15 @@ import (
 )
 
 var (
-	owner       = flag.String("owner", "istio", "Github owner or org")
-	tokenFile   = flag.String("token_file", "", "File containing Github API Access Token")
-	op          = flag.String("op", "", "Operation to be performed")
-	repo        = flag.String("repo", "", "Repository to which op is applied")
-	baseBranch  = flag.String("base_branch", "", "Branch to which op is applied")
-	refSHA      = flag.String("ref_sha", "", "Reference commit SHA used to update base branch")
-	nextRelease = flag.String("next_release", "", "Tag of the next release")
-	githubClnt  *u.GithubClient
+	owner                = flag.String("owner", "istio", "Github owner or org")
+	tokenFile            = flag.String("token_file", "", "File containing Github API Access Token")
+	op                   = flag.String("op", "", "Operation to be performed")
+	repo                 = flag.String("repo", "", "Repository to which op is applied")
+	baseBranch           = flag.String("base_branch", "", "Branch to which op is applied")
+	refSHA               = flag.String("ref_sha", "", "Reference commit SHA used to update base branch")
+	nextRelease          = flag.String("next_release", "", "Tag of the next release")
+	udpateDownloadScript = flag.Bool("update_download_script", false, "Have download script point to latest version")
```

Comment:
thanks for flipping the flag around!

typo: udpate -> update
probably should turn on the spell checker linter on this repo

Reply:
Done